include/gridtools/stencil/dump.hpp

```diff
@@ -0,0 +1,160 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#pragma once
+
+#include <ostream>
+#include <string>
+#include <typeinfo>
+
+#include <boost/core/demangle.hpp>
```

Comment:
Is it correct that this does not require a change in our Boost CMake setup? I assume so as `boost/core` sounds like it is always included, but just want to make sure.

Reply:
Yes. I checked that. This functionality remains the same since boost 1.56